var data = [{'data': '\n    <div id="btn-group">\n    <div class="btn-group btn-group-sm">\n        <a class="btn btn-primary" onclick=\'javascript:showCase(0)\'>概要{ 57.14% }</a>\n        <a class="btn btn-warning" onclick=\'javascript:showCase(4)\'>错误{ 3 }</a>\n        <a class="btn btn-danger" onclick=\'javascript:showCase(1)\'>失败{ 0 }</a>\n        <a class="btn btn-success" onclick=\'javascript:showCase(2)\'>通过{ 4 }</a>\n        <a class="btn btn-info" onclick=\'javascript:showCase(3)\'>所有{ 7 }</a>\n    </div>\n    <p></p>\n    <table id=\'result_table\' class="table table-bordered">\n        <colgroup>\n            <col align=\'left\' />\n            <col align=\'right\' />\n            <col align=\'right\' />\n            <col align=\'right\' />\n            <col align=\'right\' />\n            <col align=\'right\' />\n            <col align=\'right\' />\n        </colgroup>\n        <tr id=\'header_row\'>\n            <td>测试套件/测试用例</td>\n            <td>总数</td>\n            <td>通过</td>\n            <td>失败</td>\n            <td>错误</td>\n            <th>视图</th>\n            <th>错误截图</th>\n        </tr>\n        \n    <tr class=\'passClass\'>\n        <td>case_1_login.CaseLogin</td>\n        <td>1</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td><a href="javascript:showClassDetail(\'c1\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'pt1.1\' class=\'hiddenRow\'>\n    <td class=\'passCase\'><div class=\'testcase\'>test_login_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_pt1.1\')" >\n        通过</a>\n    <div id=\'div_pt1.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_pt1.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>pt1.1: ------开始执行用例------\ntest_login_success (case_1_login.CaseLogin) 页面中未能找到(\'id\', \'com.yzw.yunzhuang:id/st_passwordLogin\') 元素\n2020-01-08 15:37:33,307 logger.py [line:41] ERROR \'NoneType\' object has no attribute \'click\'\n------用例测试结束------\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'passClass\'>\n        <td>case_car_park.CasePack</td>\n        <td>1</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td><a href="javascript:showClassDetail(\'c2\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'pt2.1\' class=\'hiddenRow\'>\n    <td class=\'passCase\'><div class=\'testcase\'>test_mineDynamic_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_pt2.1\')" >\n        通过</a>\n    <div id=\'div_pt2.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_pt2.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>pt2.1: ------开始执行用例------\n停车场测试完成\n------用例测试结束------\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'errorClass\'>\n        <td>case_community.Casecommunity</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td>1</td>\n        <td><a href="javascript:showClassDetail(\'c3\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'ft3.1\' class=\'none\'>\n    <td class=\'errorCase\'><div class=\'testcase\'>test_search_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_ft3.1\')" >\n        错误</a>\n    <div id=\'div_ft3.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_ft3.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>ft3.1: ------开始执行用例------\ntest_search_success (case_community.Casecommunity) 页面中未能找到(\'xpath\', \'/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.view.ViewGroup/android.widget.HorizontalScrollView/android.widget.LinearLayout/android.widget.RelativeLayout[1]/android.widget.TextView\') 元素\n------用例测试结束------\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_community.py", line 19, in test_search_success\n    self.switch_navigation()\n  File "D:\\workplace\\AppiumPython\\Page\\Community_page.py", line 53, in switch_navigation\n    self.click_button(self.community_find)      # 回到发现页面\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 75, in click_button\n    self.find_element(loc).click()\nAttributeError: \'NoneType\' object has no attribute \'click\'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_community.py", line 58, in test_search_success\n    self.getScreenShot()\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 81, in getScreenShot\n    self.driver.get_screenshot_as_file(self.sh_file)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1032, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1064, in get_screenshot_as_png\n    return base64.b64decode(self.get_screenshot_as_base64().encode(\'ascii\'))\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1074, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\'value\']\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 321, in execute\n    self.error_handler.check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 29, in check_response\n    raise wde\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\nselenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'errorClass\'>\n        <td>case_defind.CaseDfind</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td>1</td>\n        <td><a href="javascript:showClassDetail(\'c4\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'ft4.1\' class=\'none\'>\n    <td class=\'errorCase\'><div class=\'testcase\'>test_defind_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_ft4.1\')" >\n        错误</a>\n    <div id=\'div_ft4.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_ft4.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>ft4.1: ------开始执行用例------\n2020-01-08 15:38:49,672 logger.py [line:35] INFO 进入动态详情\n(\'比邻小编\', \'18岁\', \'捕捉一只单身狗\', \'天气冷了，狗狗也需要穿衣服的\', \'#网红打卡#\', \'2020-01-07 15:42:32\')\n2020-01-08 15:38:53,483 logger.py [line:35] INFO (\'比邻小编\', \'18岁\', \'捕捉一只单身狗\', \'天气冷了，狗狗也需要穿衣服的\', \'#网红打卡#\', \'2020-01-07 15:42:32\')\n2020-01-08 15:38:53,483 logger.py [line:35] INFO 以上是获取动态详情内容\n2020-01-08 15:39:03,757 logger.py [line:35] INFO 点赞成功\n2020-01-08 15:39:04,608 logger.py [line:35] INFO 收藏成功\n------用例测试结束------\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_defind.py", line 46, in test_defind_success\n    self.details_filter()                           # 筛选\n  File "D:\\workplace\\AppiumPython\\Page\\unearthPage.py", line 82, in details_filter\n    self.click_button(self.details_comment_hot)\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 75, in click_button\n    self.find_element(loc).click()\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py", line 80, in click\n    self._execute(Command.CLICK_ELEMENT)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webelement.py", line 633, in _execute\n    return self._parent.execute(command, params)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 321, in execute\n    self.error_handler.check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 29, in check_response\n    raise wde\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\nselenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_defind.py", line 56, in test_defind_success\n    self.getScreenShot()\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 81, in getScreenShot\n    self.driver.get_screenshot_as_file(self.sh_file)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1032, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1064, in get_screenshot_as_png\n    return base64.b64decode(self.get_screenshot_as_base64().encode(\'ascii\'))\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1074, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\'value\']\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 321, in execute\n    self.error_handler.check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 29, in check_response\n    raise wde\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\nselenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'passClass\'>\n        <td>case_dynamic.CaseDynamic</td>\n        <td>1</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td><a href="javascript:showClassDetail(\'c5\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'pt5.1\' class=\'hiddenRow\'>\n    <td class=\'passCase\'><div class=\'testcase\'>test_mineDynamic_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_pt5.1\')" >\n        通过</a>\n    <div id=\'div_pt5.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_pt5.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>pt5.1: ------开始执行用例------\n2020-01-08 15:39:26,586 logger.py [line:35] INFO (\'发布一个测试动态\', \'谢光福\')\n2020-01-08 15:39:31,809 logger.py [line:35] INFO (\'卡住了，不知道怎么回事，是不是网络波动\', \'谢光福\')\ntest_mineDynamic_success (case_dynamic.CaseDynamic) 页面中未能找到(\'xpath\', \'/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout/android.view.ViewGroup/android.widget.FrameLayout/android.view.ViewGroup/android.support.v7.widget.RecyclerView/android.view.ViewGroup[1]\') 元素\n2020-01-08 15:39:52,932 logger.py [line:41] ERROR \'NoneType\' object has no attribute \'click\'\ntest_mineDynamic_success (case_dynamic.CaseDynamic) 页面中未能找到(\'xpath\', \'/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.RelativeLayout/android.widget.LinearLayout[2]/android.widget.RelativeLayout[2]/android.widget.LinearLayout/android.widget.ImageView\') 元素\n2020-01-08 15:40:09,609 logger.py [line:41] ERROR \'NoneType\' object has no attribute \'click\'\ntest_mineDynamic_success (case_dynamic.CaseDynamic) 页面中未能找到(\'xpath\', \'/hierarchy/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.LinearLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.FrameLayout/android.widget.RelativeLayout/android.widget.RelativeLayout/android.widget.LinearLayout[2]/android.widget.RelativeLayout[2]/android.widget.LinearLayout/android.widget.ImageView\') 元素\n2020-01-08 15:40:26,223 logger.py [line:41] ERROR \'NoneType\' object has no attribute \'click\'\n------用例测试结束------\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'passClass\'>\n        <td>case_station.CasePack</td>\n        <td>1</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td><a href="javascript:showClassDetail(\'c6\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'pt6.1\' class=\'hiddenRow\'>\n    <td class=\'passCase\'><div class=\'testcase\'>test_staion_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_pt6.1\')" >\n        通过</a>\n    <div id=\'div_pt6.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_pt6.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>pt6.1: ------开始执行用例------\n&lt;appium.webdriver.webelement.WebElement (session="acc8d732-ea75-4ba3-8454-ecb73461ad47", element="d50cf59c-daf3-43e6-811f-3b53ccc8ff69")&gt;\n&lt;appium.webdriver.webelement.WebElement (session="acc8d732-ea75-4ba3-8454-ecb73461ad47", element="91ba4935-874c-425f-9414-62047cfdce61")&gt; &lt;appium.webdriver.webelement.WebElement (session="acc8d732-ea75-4ba3-8454-ecb73461ad47", element="c4b0d463-efed-4b71-9436-ab7a87288e2b")&gt;\n------用例测试结束------\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n    <tr class=\'errorClass\'>\n        <td>case_z_logout.CaseLogout</td>\n        <td>1</td>\n        <td>0</td>\n        <td>0</td>\n        <td>1</td>\n        <td><a href="javascript:showClassDetail(\'c7\',1)">详情</a></td>\n        <td>&nbsp;</td>\n    </tr>\n\n<tr id=\'ft7.1\' class=\'none\'>\n    <td class=\'errorCase\'><div class=\'testcase\'>test_logout_success</div></td>\n    <td colspan=\'5\' align=\'center\'>\n    <!--css div popup start-->\n    <a class="popup_link" onfocus=\'this.blur();\' href="javascript:showTestDetail(\'div_ft7.1\')" >\n        错误</a>\n    <div id=\'div_ft7.1\' class="popup_window">\n        <div style=\'text-align: right; color:red;cursor:pointer\'>\n            <a onfocus=\'this.blur();\' onclick="document.getElementById(\'div_ft7.1\').style.display = \'none\' " >\n               [x]</a>\n        </div>\n        <pre>ft7.1: ------开始执行用例------\ntest_logout_success (case_z_logout.CaseLogout) 页面中未能找到(\'id\', \'com.yzw.yunzhuang:id/toolbar_set\') 元素\n------用例测试结束------\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_z_logout.py", line 19, in test_logout_success\n    self.click_setting()\n  File "D:\\workplace\\AppiumPython\\Page\\LogoutPage.py", line 24, in click_setting\n    self.click_button(self.setting_loc)\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 75, in click_button\n    self.find_element(loc).click()\nAttributeError: \'NoneType\' object has no attribute \'click\'\n\nDuring handling of the above exception, another exception occurred:\n\nTraceback (most recent call last):\n  File "D:\\workplace\\AppiumPython\\TestCase\\case_z_logout.py", line 25, in test_logout_success\n    self.getScreenShot()\n  File "D:\\workplace\\AppiumPython\\Page\\BasePage.py", line 81, in getScreenShot\n    self.driver.get_screenshot_as_file(self.sh_file)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1032, in get_screenshot_as_file\n    png = self.get_screenshot_as_png()\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1064, in get_screenshot_as_png\n    return base64.b64decode(self.get_screenshot_as_base64().encode(\'ascii\'))\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 1074, in get_screenshot_as_base64\n    return self.execute(Command.SCREENSHOT)[\'value\']\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\webdriver.py", line 321, in execute\n    self.error_handler.check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 29, in check_response\n    raise wde\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\appium\\webdriver\\errorhandler.py", line 24, in check_response\n    super(MobileErrorHandler, self).check_response(response)\n  File "C:\\Users\\yzw\\AppData\\Local\\Programs\\Python\\Python35\\lib\\site-packages\\selenium\\webdriver\\remote\\errorhandler.py", line 242, in check_response\n    raise exception_class(message, screen, stacktrace)\nselenium.common.exceptions.WebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Could not proxy command to remote server. Original error: Error: socket hang up\n\n</pre>\n    </div>\n    <!--css div popup end-->\n    </td>\n    <td>无截图</td>\n</tr>\n\n        <tr id=\'total_row\'>\n            <td>总计</td>\n            <td>7</td>\n            <td>4</td>\n            <td>0</td>\n            <td>3</td>\n            <td>&nbsp;</td>\n            <th>&nbsp;</th>\n        </tr>\n    </table>\n    </div>\n', 'duration': '0:04:19.149539', 'error': '3', 'startTime': '2020-01-08 15:37:07', 'success': '4', 'desc': '用例执行情况', 'status': '通过 4 错误 3', 'title': '自动化测试报告'}]